AppExamples/CleverDeal.React/src/Utils/utils.js

```diff
@@ -13,10 +13,6 @@ export const getEcpParam = (name: string, useSessionStorage: boolean = true): st
 
     const queryParam = new URL(window.location.href).searchParams.get(name);
 
-    if (!useSessionStorage) {
```

Comment:
`useSessionStorage` is not used anymore?

Reply:
overlooked this - `sessionStorage` is still used, but should be secondary to `queryParam`, hence the `useSessionStorage` override becomes redundant now. removed it in the latest.